src/web/handlers/GestureHandler.ts

```diff
@@ -623,8 +569,16 @@ export default abstract class GestureHandler {
     invokeNullableMethod(onGestureHandlerEvent, cancelEvent);
   }
 
-  protected transformNativeEvent() {
-    return {};
+  protected transformNativeEvent(): Record<string, unknown> {
+    // those properties are shared by most handlers and if not this method will be overriden
+    const rect = this.delegate.measureView();
+
+    return {
+      x: this.tracker.getLastAvgX() - rect.pageX,
+      y: this.tracker.getLastAvgY() - rect.pageY,
```

Comment:
Why do we switch to `pageX | pageY` from `left | top`?

Reply:
I changed `getBoundingClientRect` to use `measure` from the delegate: https://github.com/software-mansion/react-native-gesture-handler/pull/2579/files#diff-c183a9c3b38cc57a48e82ed291800d840b640919c24da2d057c24be0878695d9R60-R69. It still uses `left` and `right`, but I changed the names to be `pageX` and `pageY` as it's more explicit about what those numbers refer to (they change during scroll).

Reply:
Oh, okay. I missed the fact that after all it still uses `left` and `top` (somehow I thought it uses `pageX` and `pageY` now, even though they're part of `MouseEvent`, not `DOMRect`)